src/Sheet.php

```diff
@@ -505,7 +505,7 @@ public function fromScout(FromQuery $sheetExport, Worksheet $worksheet)
      */
     public function fromCollection(FromCollection $sheetExport)
     {
-        $this->appendRows($sheetExport->collection()->all(), $sheetExport);
+        $this->appendRows(new Collection($sheetExport->collection()->all()), $sheetExport);
```

Comment:
what makes this different from just doing `$this->appendRows($sheetExport->collection())` ?